pkg/workload/cli/run.go

```diff
@@ -428,10 +436,34 @@ func runRun(gen workload.Generator, urls []string, dbName string) error {
 	if *individualOperationReceiverAddr != "" {
 		publisher = histogram.CreateUdpPublisher(*individualOperationReceiverAddr)
 	}
-	reg := histogram.NewRegistryWithPublisher(
+
+	metricsExporter, file, err := maybeInitAndCreateExporter()
```

Comment:
`err` is effectively muted; I'm surprised this wasn't flagged by CI during linting.